src/main/java/pages/LoginPage.java

```diff
@@ -20,4 +20,11 @@ public MyAccountPage validUserLogin (String ValidEmail, String ValidPassword){
        return new MyAccountPage(driver);
    }
 
+    public LoginPage InvalidUserLogin (String InvalidEmail, String InvalidPassword){
+        typeTextInField(EMAIL_TXT, InvalidEmail);
+        typeTextInField(PASSWORD_TXT, InvalidPassword);
+        clickButton(LOGIN_Button);
+        return new LoginPage(driver);
```

Comment:
you don't have to return the loginpage, you can use return this as you are in the same page you want to return 
so replace "return new LoginPage(driver);" with "return this;"